docs/Project.toml

```diff
@@ -2,6 +2,7 @@
 CDDLib = "3391f64e-dcde-5f30-b752-e11513730f60"
 Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+Dionysos = "d92c97cf-b87d-42c1-a9c0-25df00b4d958"
```

Comment:
Remove, it's ok if it's in the `Project.toml` of your local computer but don't push it.